packages/web/src/components/Plan.tsx

```diff
@@ -105,23 +79,69 @@ export class Plan extends React.Component<{}, State> {
 
     constructor(props: {}) {
         super(props);
+        /*
+        const theData = new EventData(
+            1,
+            "Tutorial Event",
+            new Date(),
+            new Date(),
+            "This is an example event! You can add or change descriptions and titles by clicking the edit button at the side. + search help when it's implemented, etc. etc."
+        );
+
+        //const arrData = [theData];
+
+        //addEvent(arrData);
+        addTutorial(this.state.events);
+
+        console.log("The EventData is " + theData);
+        console.log("The EventData id is " + theData.id);
+        console.log("The EventData name is " + theData.name);
+        console.log("The EventData description is " + theData.description);
+        console.log("The Elements in the Array is " + this.state.events);
+        console.log("The No. of Elements is " + this.state.events.length);
+        */
     }
 
     componentDidMount(): void {
         const restoredData = restoreData();
         const restoredPlan = restorePlan();
+        const tutorialEvent = new EventData(
```

Comment:
Just use the `tutorialEvent` as the default event state when `restoredPlan` is undefined.  

`tutorialEvent` is being unused.  Your previous changes were more correct, just make a `tutorialEvent` here like you did, then set it as the event when `restoredPlan` is undefined on line 128-130.